api-reference/echo-node-api/database-api.md

```diff
@@ -1206,56 +1878,49 @@ All other accounts will be retrieved and subscribed.
         "get_full_accounts",
         [
             [
-                "1.2.10",
-                "1.2.11"
+                "1.2.10"
             ],
             "false"
         ]
     ]
 }
 ```
 
-##### Returns
+#### Notice example
 
-A map of strings from names_or_ids to the corresponding accounts.
+The notification was received by performing the operation delegating_account.
```

Comment:
I think the following should be indicated:
`The notification was received by performing the account update operation for changing delegating_account.`

Reply:
Fixed